test/extended/util/framework.go

```diff
@@ -1563,7 +1563,7 @@ func RunOneShotCommandPod(
 		}
 
 		if podHasErrored(cmdPod) {
-			return true, fmt.Errorf("the pod errored trying to run the command")
+			return true, fmt.Errorf("the pod errored trying to run the command: %#v", pod.Status)
```

Comment:
cmdPod ?

Reply:
Even if it referes to the same pod, it does sound like it would be cmdPod instead; given that it would contain the latest status (perhaps?)